pkg/testutil/sensor.go

```diff
@@ -27,31 +26,50 @@ import (
 )
 
 const (
-	commandsFileName = "commands.json"
+	// Intentionally duplicating values here since to make sure a refactoing
+	// of the paths on the sensor side won't be unnoticed.
+	artifactDirName   = "artifacts"
+	artifactDirPath   = "/opt/dockerslim/" + artifactDirName
+	commandFileName   = "commands.json"
+	sensorLogFileName = "sensor.log"
+	eventFileName     = "events.json"
 )
 
 var (
 	errNotStarted error = errors.New("test sensor container hasn't been started yet")
 )
 
+type sensorOpt func(*Sensor)
```

Comment:
will need to refactor functional options to be regular options (definitely can wait, will be able to do it later)